src/components/menu/js/menuServiceProvider.js

```diff
@@ -289,6 +289,7 @@ function MenuProvider($$interimElementProvider) {
           var handled;
           switch (ev.keyCode) {
             case $mdConstant.KEY_CODE.ESCAPE:
+            case $mdConstant.KEY_CODE.TAB:
               opts.mdMenuCtrl.close(false, { closeAll: true });
               handled = true;
```

Comment:
Looking at the logic below, when an event is marked as `handled`, it'll prevent the default action. This shouldn't be the case `TAB`, because it'll prevent people from moving forward.

Reply:
Yeah, good catch.